iotedgedev/cli.py

```diff
@@ -147,11 +153,24 @@ def add(name, template, group_id):
               required=False,
               is_flag=True,
               help="Deploy modules to Edge device using deployment.json in the config folder")
+@click.option("--file",
+              "-f",
+              "template_file",
+              default=envvars.DEPLOYMENT_CONFIG_TEMPLATE_FILE,
+              show_default=True,
+              required=False,
+              help="Specify the deployment manifest template file")
+@click.option("--platform",
+              "-P",
+              default=envvars.DEFAULT_PLATFORM,
```

Comment:
we don't have hard code default value? It seems not compatible if user use the new version tool to build solution created previously

Reply:
The default value for `DEFAULT_PLATFORM` is amd64 (loaded in https://github.com/Azure/iotedgedev/pull/358/files#diff-1fc02f4fdd45b74526b9cf54754645ceR127). It will be ignored if the user is building a legacy solution.